Gems/Terrain/Code/Source/TerrainSystem/TerrainSystem.cpp

```diff
@@ -223,6 +223,72 @@ void TerrainSystem::RoundPosition(float x, float y, float queryResolution, AZ::V
     outPosition = (normalizedPosition + AZ::Vector2(0.5f)).GetFloor() * queryResolution;
 }
 
+void TerrainSystem::InterpolateHeights(float heights[4], bool exists[4], float lerpX, float lerpY, float& outHeight, bool& outExists)
```

Comment:
`heights` and `exists` could be `const`. Would be a little cleaner to use const refs to `AZStd::array<>`.